content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -307,11 +310,14 @@ stages:
 
 ```yaml
 # dvc.lock
+schema: '2.0'
 stages:
   build@uk:
     cmd: python train.py 'uk' 3 10
     outs:
-      - model-uk.hdfs
+      - path: model-uk.hdfs
+        md5: 17b3d1efc339b416c4b5615b1ce1b97e
+        size: 2712300
```

Comment:
Is this also a change from iterative/dvc/pull/5222 or was this just wrong? Because there's another lonely output file path in line 308.

Reply:
> there's another lonely output file path in line 308.

Nvmd thats a dvc.yaml file.

Reply:
The main dvc.lock example in https://dvc.org/doc/user-guide/project-structure/pipelines-files#dvclock-file may be incomplete in `outs`. @skshetry please lmk if that's unrelated to the core change and I'll address separately (unless you'd like to fix it here of course).

Reply:
IIRC, `size` and `nfiles` were introduced in v1.9.

@jorgeorpinel, where are the examples from? Or, should I just add random values as I did above?

Reply:
Yep they are random, I mean I had a temporary project at the time and copied from there most of this I think but I didn't keep it. Feel free to add random values, and thanks!

Reply:
Added, please review. Thanks

Reply:
Oh I see it was just the `size` fields missing. Actually I think I removed those to make the docs shorter since the field is optional. Oops sorry about that... Maybe I'll remove them again (also from here). Anyway thanks again, I'll merge this soon!